### ISSUE TITLE
18_VirtualTexture breaks with dx and vk

### ISSUE THREAD

**Issue by trsh** [ConfettiFX/The-Forge#161]
![Untitled111](https://user-images.githubusercontent.com/3727523/72928135-1cf20800-3d60-11ea-9e82-887c92882df0.png)

Log:

```
'18_VirtualTexture.exe' (Win32): Unloaded 'C:\Windows\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_ce13a81bcfac4a48\nvwgf2umx.dll'
'18_VirtualTexture.exe' (Win32): Unloaded 'C:\Windows\System32\version.dll'
'18_VirtualTexture.exe' (Win32): Unloaded 'C:\Windows\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_ce13a81bcfac4a48\nvldumdx.dll'
2020-01-22 21:40:18 [MainThread     ]         direct3d12.cpp:1537  INFO| GPU[0] detected. Vendor ID: 10de, Revision ID: a2, GPU Name: NVIDIA GeForce GTX 750 Ti
2020-01-22 21:40:18 [MainThread     ]         direct3d12.cpp:1573  INFO| GPU[0] is selected as default GPU
2020-01-22 21:40:18 [MainThread     ]         direct3d12.cpp:1574  INFO| Name of selected gpu: NVIDIA GeForce GTX 750 Ti
2020-01-22 21:40:18 [MainThread     ]         direct3d12.cpp:1575  INFO| Vendor id of selected gpu: 0x10de
2020-01-22 21:40:18 [MainThread     ]         direct3d12.cpp:1576  INFO| Model id of selected gpu: 0x1380
2020-01-22 21:40:18 [MainThread     ]         direct3d12.cpp:1577  INFO| Revision id of selected gpu: 0xa2
'18_VirtualTexture.exe' (Win32): Loaded 'C:\Windows\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_ce13a81bcfac4a48\nvldumdx.dll'. Cannot find or open the PDB file.
'18_VirtualTexture.exe' (Win32): Loaded 'C:\Windows\System32\version.dll'. Cannot find or open the PDB file.
'18_VirtualTexture.exe' (Win32): Loaded 'C:\Windows\System32\DriverStore\FileRepository\nv_dispi.inf_amd64_ce13a81bcfac4a48\nvwgf2umx.dll'. Cannot find or open the PDB file.
'18_VirtualTexture.exe' (Win32): Loaded 'C:\Program Files\NVIDIA Corporation\Ansel\NvCameraWhitelisting64.dll'. Cannot find or open the PDB file.
'18_VirtualTexture.exe' (Win32): Unloaded 'C:\Program Files\NVIDIA Corporation\Ansel\NvCameraWhitelisting64.dll'
The thread 0xde4 has exited with code 0 (0x0).
'18_VirtualTexture.exe' (Win32): Loaded 'C:\Windows\System32\dxilconv.dll'. Cannot find or open the PDB file.
'18_VirtualTexture.exe' (Win32): Unloaded 'C:\Windows\System32\dxilconv.dll'
2020-01-22 21:40:19 [MainThread     ]         direct3d12.cpp:6627  INFO| Virtual Texture info: Dim 8192 x 4096 Pages 2730
The thread 0x23dc has exited with code 0 (0x0).
The thread 0x3228 has exited with code 0 (0x0).
The thread 0x8c4 has exited with code 0 (0x0).
2020-01-22 21:40:20 [MainThread     ]         direct3d12.cpp:6627  INFO| Virtual Texture info: Dim 8192 x 4096 Pages 2730
The thread 0x1b08 has exited with code 0 (0x0).
The thread 0x1458 has exited with code 0 (0x0).
The thread 0x11cc has exited with code 0 (0x0).
2020-01-22 21:40:21 [MainThread     ]         direct3d12.cpp:6627  INFO| Virtual Texture info: Dim 8192 x 4096 Pages 2730
The thread 0x1cac has exited with code 0 (0x0).
The thread 0x2fa0 has exited with code 0 (0x0).
The thread 0x2a2c has exited with code 0 (0x0).
2020-01-22 21:40:25 [MainThread     ]         direct3d12.cpp:6627  INFO| Virtual Texture info: Dim 16384 x 8192 Pages 10922
The thread 0x2dc8 has exited with code 0 (0x0).
The thread 0x1350 has exited with code 0 (0x0).
The thread 0x2f6c has exited with code 0 (0x0).
2020-01-22 21:40:27 [MainThread     ]         direct3d12.cpp:6627  INFO| Virtual Texture info: Dim 8192 x 4096 Pages 2730
The thread 0x2dc0 has exited with code 0 (0x0).
The thread 0x1414 has exited with code 0 (0x0).
The thread 0x2784 has exited with code 0 (0x0).
2020-01-22 21:40:27 [MainThread     ]         direct3d12.cpp:6627  INFO| Virtual Texture info: Dim 4096 x 2048 Pages 682
The thread 0x1100 has exited with code 0 (0x0).
The thread 0x6dc has exited with code 0 (0x0).
The thread 0x350 has exited with code 0 (0x0).
2020-01-22 21:40:28 [MainThread     ]         direct3d12.cpp:6627  INFO| Virtual Texture info: Dim 4096 x 4096 Pages 1365
The thread 0x2c1c has exited with code 0 (0x0).
The thread 0x332c has exited with code 0 (0x0).
The thread 0x2a24 has exited with code 0 (0x0).
2020-01-22 21:40:28 [MainThread     ]         direct3d12.cpp:3761  INFO| Descriptor (uSampler0) : User specified Static Sampler
2020-01-22 21:40:28 [MainThread     ]         direct3d12.cpp:3761  INFO| Descriptor (uSampler0) : User specified Static Sampler
2020-01-22 21:40:28 [MainThread     ]         direct3d12.cpp:3796  INFO| Descriptor (uniformBlock_rootcbv) : User specified D3D12_ROOT_PARAMETER_TYPE_CBV
2020-01-22 21:40:28 [MainThread     ]         direct3d12.cpp:3937  INFO| Root constant (uRootConstants) has (6) 32 bit values. It is recommended to have root constant number less or equal than 13
2020-01-22 21:40:28 [MainThread     ]         direct3d12.cpp:3761  INFO| Descriptor (uSampler) : User specified Static Sampler
'18_VirtualTexture.exe' (Win32): Loaded 'C:\Windows\System32\dcomp.dll'. Cannot find or open the PDB file.
'18_VirtualTexture.exe' (Win32): Loaded 'C:\Windows\System32\dxilconv.dll'. Cannot find or open the PDB file.
D3D12 ERROR: ID3D12Device::CreatePixelShader: Shader uses tiled resources shader ops, but the device does not support this. To check for support, check device caps via the CheckFeatureSupport() API [ STATE_CREATION ERROR #93: CREATEPIXELSHADER_INVALIDSHADERBYTECODE]
Exception thrown at 0x00007FF8BFC09159 in 18_VirtualTexture.exe: Microsoft C++ exception: _com_error at memory location 0x0000007F6C2FB6F0.
```

**Comment by wolfgangfengel** [comment]
We will look into this. My guess is your NVIDIA 750 TI doesn't support it ... in that case we need to catch this.

**Comment by trsh** [comment]
Thank you

**Comment by wolfgangfengel** [comment]
GPU didn't support feature. Fixed with release 1.40.
